test/integration/integration_test.py

```diff
@@ -0,0 +1,144 @@
+## The integration tests consists of a back and forth between python (which we use to create and validate
+## image queries) and bash (which we use to check deployments are properly rolled out)
+## This file contains all the modes that we use for integeration testing.
+## Modes:
+## - Create the integration test detector
+## - Submit the initial dog/cat image query to the edge, expect low confidence
+## - Train the edge model by submitting image queries to the cloud.
+## - Submit the final dog/cat image query to the edge, expect high confidence
+
+import argparse
+import random
+import time
+
+from groundlight import Groundlight, GroundlightClientError
+from model import Detector
+
+NUM_IQS_TO_IMPROVE_MODEL = 10
+ACCETABLE_TRAINED_CONFIDENCE = 0.8
```

Comment:
```suggestion
ACCEPTABLE_TRAINED_CONFIDENCE = 0.8
```
And wherever this value is used